resources/windows/windows.json

```
@@ -128,10 +128,9 @@
       "type": "shell"
     },
     {
-      "scripts": [
-        "scripts/windows/configs/ssh-auth.ps1"
-      ],
-      "type": "powershell"
+      "destination": "/Users/vagrant/.ssh/authorized_keys",
```

I imagine this should be:

```suggestion
      "destination": "C:/Users/vagrant/.ssh/authorized_keys",
```